### ISSUE TITLE
"OK Google everywhere" doesnt work...

### ISSUE THREAD

**Issue by natri23** [Sabissimo/AssistantEnabler#12]
I'm using a Galaxy J7 with android M and latest Google. After the recent update to Google 6.10 and Assistant Enabler 4.0, "OK Google everywhere" doesnt work anymore. I can only activate it from the home screen.
I've tried delete Google cache, uninstall/reinstall, moved apk to /system, downgraded it to latest stable 6.9/6.8...same thing. It showed my device was not supported for ok google everywhere or something. And I cant train voice model anymore. It was working fine two days ago (I was able to activate it everywhere except when screen off)

![screenshot_20170107-174406](https://cloud.githubusercontent.com/assets/1286842/21743567/e53f4fe2-d504-11e6-9326-34f0d01d2bab.png)

**Comment by natri23** [comment]
![screenshot_20170107-182631](https://cloud.githubusercontent.com/assets/1286842/21743667/1f2aa18c-d507-11e6-8d2c-6991b8977080.png)

**Comment by natri23** [comment]
Also doesn't work with the old 3.2 version...

**Comment by Sabissimo** [comment]
Does "OK Google" work on your device with module disabled?

**Comment by natri23** [comment]
Yes, it does work with Google Now but only from the home screen and no options to retrain or enable "any time" or even "from any apps"

I think they add a device check or something in other Google apps (like Google Maps...) that prevent OK Google anytime

![screenshot_20170107-225427](https://cloud.githubusercontent.com/assets/1286842/21745560/eef4fa44-d52e-11e6-9570-f2d927f07d47.png)

**Comment by natri23** [comment]
Long-press on Home button does work though...

**Comment by natri23** [comment]
Ok, I factory reset and Assistant now works on any screen but not when screen off not charging
